src/kOS/Screen/TermWindow.cs

```diff
@@ -803,13 +812,10 @@ void TerminalGui(int windowId)
             Rect rasterBarsButtonRect = new Rect(10, WindowRect.height - 42, 85, 18);
             Rect brightnessRect = new Rect(3, WindowRect.height - 100, 8, 50);
             Rect brightnessButtonRect = new Rect(1, WindowRect.height - 48, brightnessButtonImage.width, brightnessButtonImage.height);
-            Rect fontWidthLabelRect = new Rect(35, WindowRect.height-28, 20, 10);
-            Rect fontWidthLessButtonRect = new Rect(65, WindowRect.height-28, 10, 10);
-            Rect fontWidthMoreButtonRect = new Rect(90, WindowRect.height-28, 10, 10);
-            Rect fontHeightButtonRect = new Rect(140, WindowRect.height-32, fontHeightButtonImage.width, fontHeightButtonImage.height);
-            Rect fontHeightLabelRect = new Rect(160, WindowRect.height-28, 20, 10);
-            Rect fontHeightLessButtonRect = new Rect(185, WindowRect.height-28, 10, 10);
-            Rect fontHeightMoreButtonRect = new Rect(210, WindowRect.height-28, 10, 10);
+            Rect fontHeightButtonRect = new Rect(30, WindowRect.height-33, fontHeightButtonImage.width, fontHeightButtonImage.height);
+            Rect fontHeightLabelRect = new Rect(45, WindowRect.height-33, 20, 13);
+            Rect fontHeightLessButtonRect = new Rect(75, WindowRect.height-37, 12, 13);
+            Rect fontHeightMoreButtonRect = new Rect(100, WindowRect.height-37, 12, 13);
```

Comment:
While I was working on this, I noticed that the various ``Rect``s for the font width button were still here, even though the code that used them had been deleted when we removed the ability to set the font width.

I moved the font height buttons over to where the font width ones were, which gives more space to handle drawing the terminal properly when the terminal size is tiny (at small sizes, the font height buttosns had been getting drawn on top of some of the other widgets before.)